xxxx-release-and-versioning.md

```diff
@@ -0,0 +1,87 @@
+| ------------------- | -------- |
+| Linting             | **Yes**  |
+| Unit testing        | **Yes**  |
+| Integration testing | **Yes**  |
+| End to end testing  | No       |
+| Migration testing   | No       |
+
+The development release will be provided as a binary release and a Docker image based on Alpine Linux. The Docker image will be tagged with commit ID and will have a `:dev` tag to refer to the latest development release.
+
+### Branching Model
+
+The branching model will utilize a `master` branch, a stable version branch and a feature branch. To demostrate:
+
+```
+feature2    -o---------o----------o
+                                   \
+                                    \
+feature1    -o                       \
+              \                       \
+               v                       v
+master      ----o-----------------------o-------------------------------->
+                             \                           \
+                              v                           \
+stable-1.0                     o-----------o----o----------^------------->
+                            v1.0.0           v1.0.1         \
+                                                             v
+stable-1.1                                                    o---------->
+                                                            v1.1.0
+```
+
+#### Feature Branches
+
+Feature branches, such as `feature2` and `feature1`, are development branches that implements new feature or change existing functionality. These branches may be long-running and should be merged to `master` only when the feature is considered stable enough for public consumption. Feature branches should regularly sync up with master to avoid too much code diversion. Feature branches may have its own public testing environment for each branch with its data copied from staging on each deploy.
+
+#### `master` branch
+
+The `master` branch is the branch that development release will be based on. While we required that feature branch must be stable enough before it could be merged into `master`, the `master` branch may still not stable enough to use in production environment. The `master` branch is where a stable branch will be branched from when it is considered stable enough. Before branching into stable, `master` may undergo a period of feature freeze, in which new features are not allowed to be merged onto the master branch, and only stability or bugfixes are accepted. The master branch will be deployed automatically to the OmiseGO staging environment for internal and public testing.
+
+#### Stable version branches
+
+Stable version branches such as `stable-1.0` and `stable-1.1`, are branches that correspond to the `MAJOR.MINOR` release versions. The branch is branched from `master` when the `master` branch is considered enable enough. Stable version branches are solely for maintenance purpose and will not accept any new features except for bug or security fixes.
```

Comment:
Suggest naming version branches as  `v1.0`, `v1.1`, etc. instead, Similar to Elixir and quite a few other open source projects.